test/strategy.coffee

```diff
@@ -129,7 +134,7 @@ describe 'EVE Online OAuth Strategy', ->
         @oAuth2VerifyDone)
 
     it 'translates passport-oauth2 verifications', ->
-      @verify.should.be.calledWith(@profile, @oAuth2VerifyDone)
+      @verify.should.be.calledWith('access token', 'refresh token', @profile, @oAuth2VerifyDone)

```

Comment:
That's the call, but where are "access token" and "refresh token" ever passed in anywhere but here?